appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/annotation/introspection/ConstantPoolInfo.java

```diff
@@ -122,6 +122,17 @@ public boolean containsAnnotation(int constantPoolSize, final ByteBuffer buffer)
                     buffer.getShort();
                     buffer.getShort();
                     break;
+                case METHODHANDLE:
```

Comment:
Copyright strings should be added:
// Portions Copyright [2016] [C2B2 Consulting Limited and/or its affiliates]